modules/oe_theme_content_news/oe_theme_content_news.module

```diff
@@ -75,3 +75,19 @@ function oe_theme_content_news_node_view_alter(array &$build, EntityInterface $e
     }
   }
 }
+
+/**
+ * Implements template_preprocess_node() for the News node type.
+ */
+function oe_theme_content_news_preprocess_node__oe_news__teaser(array &$variables): void {
+  /** @var \Drupal\node\NodeInterface $node */
+  $node = $variables['node'];
```

Comment:
Maybe we could add the sticky in all nodes so we avoid repeating the same thing over and over.